features/follow-account.feature

```diff
@@ -13,3 +13,16 @@ Feature: Follow accounts from their handle
   Scenario: We cannot follow ourselves
     Given we follow "Us"
     Then the request is rejected with a 400
+
```

Comment:
Should we add a scenario for being unfollowed by somebody else?

Reply:
Dun!